sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderThread.kt

```diff
@@ -435,24 +437,9 @@ internal class MapboxRenderThread : Choreographer.FrameCallback {
       renderEvent.runnable?.let {
         renderEventQueue.add(renderEvent)
       }
-      // in case of native Mapbox events we schedule render event only when we're fully ready for render
-      if (renderEvent.eventType == EventType.SDK) {
-        if (renderThreadPrepared) {
-          postPrepareRenderFrame()
-        }
-      } else {
-        postPrepareRenderFrame()
-      }
+      postPrepareRenderFrame()
```

Comment:
Here we previously were skipping render events if renderThreadPrepared was false

Reply:
I assume it was just incorrect. This was the actual reason why non-render events (e.g. 3d puck model load event) that may arrive after surface destroy (hence renderThreadPrepared == false) were just skipped.

Reply:
@kiryldz there were no other reasons, right?

Reply:
UPD - this was about non-render events, of course. 
For the render events it might be performance optimization indeed, to avoid making `post`, that doesn't seem that impactful anyway. 
@kiryldz could you bring more context here?

Reply:
seems fine to avoid making redundant post, runnable is saved in the queue anyway before

Reply:
> seems fine to avoid making redundant post, runnable is saved in the queue anyway before

Agree, postRender will anyway be triggered when rendering will continue

Reply:
Updated